docs/obs_ops.rst

```diff
@@ -13,4 +13,17 @@ obs_ops.sources
 Module containing functions for making a SSO footprint.
 
 .. automodule:: sotodlib.obs_ops.sources
-   :members:
\ No newline at end of file
+   :members:
+   
+obs_ops.splits
+==============
+
+Module containing functions for making splits of the data
+
+.. automodule:: sotodlib.obs_ops.splits
+    :members:
+
+Reference
+---------
+
+.. autofunction:: sotodlib.obs_ops.splits.det_splits_relative
```

Comment:
You can delete these lines, the line above that calls `:members:` means that it will render the docstrings for all of the functions within the `obs_ops.splits` module so currently you're duplicating the docstring for that fn in two places:
<img width="806" alt="Screenshot 2024-05-01 at 10 21 02 AM" src="https://github.com/simonsobs/sotodlib/assets/25913911/ecd18b7f-3d92-4f2c-afac-4083c63fd116">